src/components/Cat.js

```diff
@@ -1,10 +1,13 @@
+import { useDispatch } from 'react-redux';
 import Header from './Header';
+import { checkStatus } from '../redux/categories/categories';
 
 function Cat() {
+  const dispatch = useDispatch();
   return (
     <div className="Cat">
       <Header />
-      <button type="button">Check status</button>
+      <button type="button" onClick={() => { dispatch(checkStatus()); }}>Check status</button>
     </div>
   );
 }
```

Comment:
- [ ] (Optional) Even though it is not mandated, it would be nice to access the store and retrieve the value for categories when the `checkStatus` action creator is dispatched. You can use the `useSelector` the same way you did in the App component. Then, you can display the string `'Under Construction'` when the `check status` button is clicked. 👍